sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationManager.kt

```diff
@@ -155,6 +155,17 @@ interface ViewAnnotationManager {
    */
   fun getViewAnnotationUpdateMode(): ViewAnnotationUpdateMode
 
+  /**
+   * Retrieve added [View] with their [ViewAnnotationOptions]. This will return
+   * fresh copies of [View] and associated [ViewAnnotationOptions] every time.
```

Comment:
IMHO wording is a bit confusing, we won't copy actual `views`, right.

Reply:
yeah better wording can be `This will return Map of [View] and associated [ViewAnnotationOptions] every time.`, wdyt?